SetupDataPkg/ConfProfileMgrDxe/UnitTest/ConfProfileMgrDxeUnitTest.c

```diff
@@ -1245,13 +1081,6 @@ UnitTestingEntry (
   AddTestCase (ConfProfileMgrDxeTests, "ConfProfileMgrDxe should write the cached profile and reset", "ConfProfileMgrDxeShouldWriteCachedProfileAndReset", ConfProfileMgrDxeShouldWriteCachedProfileAndReset, NULL, NULL, NULL);
   AddTestCase (ConfProfileMgrDxeTests, "ConfProfileMgrDxe should write the generic profile and reset", "ConfProfileMgrDxeShouldWriteGenericProfileAndReset", ConfProfileMgrDxeShouldWriteGenericProfileAndReset, NULL, NULL, NULL);
 
-  //
-  // ConfProfileMgrDxe in MfgMode
-  //
-  AddTestCase (ConfProfileMgrDxeTests, "ConfProfileMgrDxe should use the retrieved active profile in Mfg Mode", "ConfProfileMgrDxeShouldUseRetrievedProfileMfgMode", ConfProfileMgrDxeShouldUseRetrievedProfileMfgMode, NULL, NULL, NULL);

```

Comment:
I think we should still keep the unit tests (if not adding more) to verify the mfg mode still works as expected.

Reply:
Keep the unit tests for mfg mode.